  0%|          | 0/62 [00:00<?, ?it/s]  0%|          | 0/62 [00:14<?, ?it/s]
Traceback (most recent call last):
  File "/lustre/home/acct-stu/stu312/fy/Project/av_scene_classify/train.py", line 157, in <module>
    training_loss.append(train(epoch))
  File "/lustre/home/acct-stu/stu312/fy/Project/av_scene_classify/train.py", line 98, in train
    logit = model(audio_feat, video_feat, 'late')
  File "/lustre/home/acct-stu/stu282/Tools/miniconda3/envs/py3.10.11/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/lustre/home/acct-stu/stu282/Tools/miniconda3/envs/py3.10.11/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/lustre/home/acct-stu/stu312/fy/Project/av_scene_classify/models.py", line 52, in forward
    output = (audio_out + video_out) / 2
NameError: name 'video_out' is not defined. Did you mean: 'video_feat'?
Traceback (most recent call last):
  File "/lustre/home/acct-stu/stu312/fy/Project/av_scene_classify/evaluate.py", line 47, in <module>
    model.load_state_dict(torch.load(
  File "/lustre/home/acct-stu/stu282/Tools/miniconda3/envs/py3.10.11/lib/python3.10/site-packages/torch/nn/modules/module.py", line 2153, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for MeanConcatDense:
	Missing key(s) in state_dict: "audio_output.0.weight", "audio_output.0.bias", "audio_output.1.weight", "audio_output.1.bias", "video_output.0.weight", "video_output.0.bias", "video_output.1.weight", "video_output.1.bias". 
